packages/block-library/src/cover/edit/index.js

```diff
@@ -97,6 +98,8 @@ function CoverEdit( {
 		tagName: TagName = 'div',
 	} = attributes;
 
+	const [ averageMediaColor, setAverageMediaColor ] = useState( undefined );
```

Comment:
Because you're storing the `averageMediaColor` with `useState`, when you reload the page, the state is lost and changing the `dimRatio`, for example, results in incorrect calculations since `averageMediacolor` is then `undefined`.

Reply:
Oh no another effect to write.

Reply:
Or a function to memoize :)